modules/nf-core/melon/meta.yml

```
@@ -0,0 +1,61 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "melon"
+description: Melon command for taxonoming profiling
+keywords:
+  - profile
+  - metagenomics
+  - melon
+  - classification
+tools:
+  - "melon":
+      description: "Melon: metagenomic long-read-based taxonomic identification and
+        quantification using marker genes"
+      homepage: "https://github.com/xinehc/melon"
+      documentation: "https://github.com/xinehc/melon"
+      tool_dev_url: "https://github.com/xinehc/melon"
+      doi: "10.1186/s13059-024-03363-y"
+      licence: ["MIT"]
+      identifier: ""
+input:
+  - - meta:
+        type: map
+        description: |
+          Groovy Map containing sample information
+          e.g. `[ id:'sample1', single_end:true ]`.
+    - reads:
+        type: file
+        description: Quality-controlled long reads.
+        pattern: "*.{fa,fasta,fq,fastq,fq.gz,fastq.gz}"
```

```suggestion
        pattern: "*.{fa,fasta,.fna,.fas,fq,fastq,fq.gz,fastq.gz}"
```

And the gzipped versions

---

```suggestion
        pattern: "*.{fa,fasta,fq,fastq}{,.gz}"
```
To cover all expansions.